feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt

```diff
@@ -100,8 +100,10 @@ fun SearchRoot(
 
     val keyboardController = LocalSoftwareKeyboardController.current
 
-    if (categoryId != null) {
-        viewModel.onCategorySelected(categoryId)
+    LaunchedEffect(categoryId) {
+        if (categoryId != null) {
+            viewModel.onCategorySelected(categoryId)
```

Comment:
Sorry, can we use initialCategoryId instead of categoryId? Because it is used only for the initial state.